pyproject.toml

```diff
@@ -143,35 +143,50 @@ tracker = "https://github.com/numpy/numpy/issues"
 # `tools/ci/cirrus_wheels.yml`.
 build-frontend = "build"
 skip = "cp36-* cp37-* cp-38* pp37-* *-manylinux_i686 *_ppc64le *_s390x"
-build-verbosity = "3"
 before-build = "bash {project}/tools/wheels/cibw_before_build.sh {project}"
 # The build will use openblas64 everywhere, except on arm64 macOS >=14.0 (uses Accelerate)
-config-settings = "setup-args=-Duse-ilp64=true setup-args=-Dallow-noblas=false build-dir=build"
```

Comment:
@mattip why did you remove `use-ilp64` here? It looks like it switched all the wheels to LP64. Log extracts:
```
# manylinux x86-64:
  Run-time dependency scipy-openblas found: YES 0.3.26
  Message: BLAS symbol suffix: 
```

```
#  Accelerate:
Run-time dependency accelerate found: YES
Message: BLAS symbol suffix: $NEWLAPACK
```

Reply:
I moved the macros to the pkg-config file [here](https://github.com/MacPython/openblas-libs/blob/d65af288e71e4a2099120d18a9cfbaabd77b008d/local/scipy_openblas64/__init__.py#L73) so it is not needed for the scipy-openblas build. I guess it is important to leave it in for diagnostics and for other blas builds though, right?

Reply:
yes indeed